### ISSUE TITLE
Airflow run dag with config 

### ISSUE THREAD

**Issue by daBlesr** [getindata/kedro-airflow-k8s#136]
Hello!

Is there a way to access airflow config parameters during a dag run? For example, like specified on this answer on stack overflow [Accessing configuration parameters passed to Airflow through CLI](https://stackoverflow.com/a/53606745/1094516).

If this is not possible, what code should I look at to allow for this change, so I can create a PR?

**Comment by szczeles** [comment]
Take a look here: https://github.com/getindata/kedro-airflow-k8s/blob/0.8.0/kedro_airflow_k8s/config.py#L93 - this parameter allows you to specify list of airflow variables that are pushed as Kedro parameters

**Comment by daBlesr** [comment]
That works indeed for fixed variables in Airflow. In this case, I'd like to run a DAG with specific variables only for that single DAG run. That is a different thing and is not picked up.

**Comment by szczeles** [comment]
That's right, the config provided during "trigger dag" operation is not used. If you have free time, feel free to create a PR. I'd plug here: https://github.com/getindata/kedro-airflow-k8s/blob/0.8.0/kedro_airflow_k8s/airflow_dag_template.j2#L303 with:
* static list of possible configs provided in the config file
* for loop in the template, using [dag_run.conf](https://forum.astronomer.io/t/retrieving-dag-run-time-config-values/1492/2) to get the values

It's not the best way, as the user can't explicitly override any parameter without the static list of possible lookups. Maybe during implementation you will find a better solution!
